### DISCUSSION TITLE
Keymaps not working

### DISCUSSION THREAD

**Discussion by Revoid07** [PlayCover/PlayCover#1105] [Q&A] [answered]
I found a configuration profile in the profiles tab in settings and wasn't sure what it did, so I deleted it. But now whenever I try and open Genshin Impact, the keymap doesn't work. The mouse will still be mapped, but it will only click, and pressing option doesn't exit the configured map. I tried reinstalling playcover, but it didn't reinstall the profile. How can I reinstall the profile so my mappings work?

**Comment by MoeNyanSakura** [comment, marked as answer]
Same as #1097 and #1048

**Reply by TheMoonThatRises** [reply]
To clarify, install the nightly version of PlayCover.
